packages/apollo-voyager-server/package.json

```diff
@@ -20,7 +20,7 @@
     "compile": "tsc --build tsconfig.json",
     "watch": "tsc --build tsconfig.json --watch",
     "compile:clean": "tsc --build tsconfig.json --clean",
-    "test": "ava '*.test.ts' '**/*.test.ts'"
+    "test": "ava -v '*.test.ts' '**/*.test.ts'"
```

Comment:
Good idea having `-v` in package.json scripts.
But, I think we should do the same in other packages too, to be consistent.